framework/packages/gov-ownable/derive/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "cw-gov-ownable-derive"
+version = "0.1.0"
```

Comment:
Do we want to maintain this derive repo? I would happily throw it :)

Reply:
We have to keep it (imports are different)